deploy/rbd/helm/Chart.yaml

```diff
@@ -0,0 +1,12 @@
+apiVersion: v1
+appVersion: "0.3.0"
+description: Container Storage Interface (CSI) driver, provisioner, and attacher for Ceph RBD
+name: ceph-csi-rbd
+version: 0.2.0
```

Comment:
0.2.0 is lofty start :)

Reply:
I think we can set any version, 0.2.0 is from my own usage.  I'd say it should be 1.0 when it is updated for the 1.0 CSI driver.

Reply:
let's do that. Would you please set 0.1 for now and when we merge csi-v1.0 branch, we bump it up.